spec/models/invoice_spec.rb

```diff
@@ -23,4 +23,10 @@
       expect(invoice.status).to eq "shipped"
     end
   end
+
+  describe ".revenue_by_day" do
+    it "responds to revenue_by_day"do
+      expect(Invoice).to respond_to(:revenue_by_day)
+    end
+  end
 end
```

Comment:
I like this test. Let's make sure we add these in for all of our other business intelligence methods.